.github/CONTRIBUTING.md

```diff
@@ -16,56 +16,61 @@ One important thing distinction is that we are building the Polkadot Runtime Env
 
 ## Contribution Steps
 
-**1. Set up gossamer following the instructions our [README](https://github.com/ChainSafe/gossamer/blob/development/README.md).**
+**1. Fork the gossamer repo.**
 
-**2. Fork the gossamer repo.**
+**2. Create a local clone of gossamer.**
 
-**3. Create a local clone of gossamer.**
+```
+go get -u github.com/ChainSafe/gossamer
+cd $GOPATH/src/github.com/ChainSafe/gossamer
+git init
+```
+You may encounter a `package github.com/ChainSafe/gossamer: no Go files in ...` message when doing `go get`. This is not an error, since there are no go files in the project root.
 
-**4. Link your local clone to the fork on your Github repo.**
+**3. Link your local clone to the fork on your Github repo.**
 
 ```
-$ git remote add your-gossamer-repo https://github.com/<your_github_user_name>/gossamer.git
+$ git remote add your-gossamer-repo https://github.com/your_github_user_name/gossamer.git
```

Comment:
could you re-add the `<` `>` around `your_github_user_name`?

Reply:
so sorry

Reply:
no worries :)